### PR TITLE
Added new animals to found animals guide

### LINKED ISSUE #676: [Idea]: Found animal guides for rabbits, foxes, skunks and opossums
### Describe your idea / enhancement

Add more "found an animal" guides for these animals:

- [ ] rabbits
- [ ] foxes
- [ ] skunks
- [ ] opossums

Inspo:

- https://www.humanesociety.org/resources/found-orphaned-or-injured-baby-wild-animal#rabbits
- https://spca.bc.ca/faqs/i-found-baby-rabbit/

### How would this idea / enhancement be beneficial?

n/a

### PR DESCRIPTION
**Pull request by loganrx8**
## Describe your changes

Added four new animals to found animals guide
- Rabbits
- Foxes
- Skunks
- Opossums

closes #676

### CODE REVIEW

File: apps/website/src/data/found-animal.ts [hunk 1]
```diff
@@ -360,6 +360,176 @@ const data: FoundAnimalFlow = {
...
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Fox",
+      flow: {
+        prompt: ["Does the fox appear to be sick or injured,?"],
```
Comment (pjeweb) [inline apps/website/src/data/found-animal.ts:409]:
```suggestion
        prompt: ["Does the fox appear to be sick or injured?"],
```

File: apps/website/src/data/found-animal.ts [hunk 2]
```diff
@@ -360,6 +360,176 @@ const data: FoundAnimalFlow = {
...
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "Fox kits will often appear unsupervised for long periods of time while their parents are out hunting. They will play like puppies around the den until the parents decide they are old enough to go hunting. Then they will disappear. Observe the kits from a distance; if they seem energetic and healthy, leave them alone.",
+                      ...macros.general.rehab(
+                        "the baby fox",
+                        "If they appear sickly or weak, or if you have reason to believe both parents are dead.",
```
Comment (pjeweb) [inline apps/website/src/data/found-animal.ts:431]:
```suggestion
                        "If they appear sickly or weak, or if you have reason to believe both parents are dead",
```

File: apps/website/src/data/found-animal.ts [hunk 3]
```diff
@@ -360,6 +360,176 @@ const data: FoundAnimalFlow = {
...
+              prompt: [
+                "Have you found a baby rabbit that's alone with no mother in sight?",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "Rabbits at least four inches long with open eyes and erect ears that can hop well are independent from their mother and should be allowed to fend for themselves. Uninjured baby rabbits should also be left alone. Although they may look abandoned because their mom isn't around, they are probably fine: Mother rabbits visit their dependent babies only a few times a day to avoid attracting predators.",
+                      "If you are still concerned you can monitor them from a distance over a period of 24 hours, Keep pets and children away from the area, avoid touching the babies",
```
Comment (pjeweb) [inline apps/website/src/data/found-animal.ts:386]:
```suggestion
                      "If you are still concerned you can monitor them from a distance over a period of 24 hours, keep pets and children away from the area, avoid touching the babies.",
```

File: apps/website/src/data/found-animal.ts [hunk 4]
```diff
@@ -360,6 +360,176 @@ const data: FoundAnimalFlow = {
...
+    },
+    {
+      name: "Opossum",
+      flow: {
+        prompt: ["Does the opposum appear to be sick or injured?"],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: macros.general.rehab("the rabbit"),
```
Comment (pjeweb) [inline apps/website/src/data/found-animal.ts:498]:
```suggestion
              prompt: macros.general.rehab("the opossum"),
```

**Review by pjeweb — CHANGES_REQUESTED** [review]
A few small fixes. In general how would you feel about giving sizes in metric too? (cm for inches etc.)

**Comment by loganrx8** [comment]
> A few small fixes. In general how would you feel about giving sizes in metric too? (cm for inches etc.)

It's fine with me 👍

**Review by loganrx8 — COMMENTED** [review]
Added sizes in metric.

**Review by loganrx8 — COMMENTED** [review]
Fixed spelling mistakes

**Review by pjeweb — APPROVED** [review]

**Comment by pjeweb** [comment]
✅ reviewed by staff

**Pull request merged**